src/style.css

```diff
@@ -1 +1,37 @@
 /* styles go here */
+.fieldset input {
```

Comment:
1) style classes instead of tags
2) you can simplify your styles by adding class to a label (or another `<input>`/`<select>` wrapper an use something like this
```
.fieldset:not(:last-child) {
  margin-bottom: {{your_value}};
}

.label:not(:last-child) {
  margin-bottom: {{your_value}};
}
```

Reply:
not fixed
![image](https://user-images.githubusercontent.com/98764641/229069248-42e44903-0b0f-49cc-bf03-084ac25f5e57.png)